kurtosis-devnet/op-program-svc/fs_test.go

```diff
@@ -134,7 +134,6 @@ func TestProofFileSystem(t *testing.T) {
 		for _, file := range files {
 			if file.Name() == "info.json" {
 				foundInfoJson = true
-				break
```

Comment:
Removed by mistake?

Reply:
This is so weird, my local branch doesn't have that removal. I have no idea what I pushed 🤔 

Regardless, yes you're absolutely right, that code is pretty stupid :)